chartfx-dataset/src/main/java/de/gsi/dataset/spi/DataSetBuilder.java

```diff
@@ -26,9 +32,11 @@
  * <li>dim:3+, useFloat=false, errors=false: {@link MultiDimDoubleDataSet}
  * <li>All other combinations throw a {@code UnsupportedOperationException}
  * </ul>
+ * 
  * @author Alexander Krimm
  */
 public class DataSetBuilder {
+    static final Logger LOGGER = LoggerFactory.getLogger(DataSetBuilder.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5594966)